tests/Tests/Api/PractitionerApiTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+namespace OpenEMR\Tests\Api;
+
+use PHPUnit\Framework\TestCase;
+use OpenEMR\Tests\Api\ApiTestClient;
+use OpenEMR\Tests\Fixtures\PractitionerFixtureManager;
+
+/**
+ * Practitioner API Endpoint Test Cases.
+ * NOTE: currently disabled (by naming convention) until work is completed to support running as part of Travis CI
+ * @coversDefaultClass OpenEMR\Tests\Api\ApiTestClient
```

Comment:
Guessing the above 2 lines can be removed

Reply:
Definitely! 11 LN can be removed can't guess the second one?

Reply:
oops, just meant line 11 :)